packages/monitoring-system/data-processor/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "data-processor",
+  "version": "1.0.0",
+  "description": "Data ingestion and transformation system for GitHub Repo Automation Bots",
+  "scripts": {
+    "compile": "tsc -p .",
+    "pretest": "npm run compile",
+    "fix": "gts fix",
+    "lint": "gts check",
+    "test": "cross-env LOG_LEVEL=fatal c8 mocha --exit ./build/test",
+    "start": "node ./build/src/task-service.js",
+    "docker-start-local": "docker run --network=host data-processor:1.0",
+    "docker-build-local": "docker build --tag data-processor:1.0 .",
+    "local-docker": "npm run compile && npm run docker-build-local && npm run docker-start-local"
+  },
+  "repository": "https://github.com/googleapis/repo-automation-bots.git",
+  "homepage": "https://github.com/googleapis/repo-automation-bots",
+  "bugs": "https://github.com/googleapis/repo-automation-bots/issues",
+  "main": "./build/src/task-service.js",
+  "author": "asonawalla@google.com",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "express": "^4.17.1",
+    "firebase-admin": "^9.0.0"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.2.11",
```

Comment:
Let's use the built in `assert` module please :)

Reply:
For the HTTP stuff, supertest is what we generally use here

Reply:
I believe the chai agent extends supertest but I can definitely swap it out!